pytest_minimal.ini

```diff
@@ -1,3 +1,6 @@
 [pytest]
-addopts = --ds=main.settings
+addopts = --ds=main.settings --reuse-db
```

Comment:
Just curious what the changes in this file are for, do they fix other issues?

Reply:
I just added the things that are in the main `pytest.ini` file but aren't present here, because I wanted to be able to run the tests without all the coverage related stuff and the linter and all that running every time